tests/System.Text.JsonLab.Tests/JsonObjectTests.cs

```diff
@@ -212,8 +212,9 @@ public void ParseJson(bool compactData, TestCaseType type, string jsonString)
 
                 if (type == TestCaseType.Json400KB)
                 {
-                    expectedString = ReadJson400KB(jtoken);
-                    actualString = ReadJson400KB(obj);
+                    // TODO: Investigate why 1 + 75486/(double)100000 == 1.7548599999999999 instead of just 1.75846
```

Comment:
@tannergooding, is this expected behavior? Shows up in 3.0 only (not in netfx/.NET Core 2.1):
```C#
double x = 1 + 75486/(double)100000;
Console.WriteLine(x);   // 1.7548599999999999
```

Reply:
Yes. The raw bits for `1 + 75486 / (double)100000` are `0x3FFC13E81450EFDC`
The raw bits for `1.75486` are `0x3FFC13E81450EFDD`

In full framework, `ToString()` returned `1.75486` for both of them, which meant you couldn't differentiate.

In core, we return the shortest roundtrippable string, which allows them to always be differentiated.